### PR TITLE
updates to A-Frame 6.1

### PR DESCRIPTION
**Pull request by caseyyee**
Also see conversation in https://github.com/aframevr/a-saturday-night/pull/117 

@fernandojsg

### CODE REVIEW

File: src/ui2d.js [hunk 1]
```diff
@@ -16,9 +16,9 @@ window.addEventListener('load', function (event) {
     instructions.classList.remove('hide');
   }
 
-  sceneEl.addEventListener('enter-vr', function () {
-    instructions.classList.add('hide');
-  });
+  // sceneEl.addEventListener('enter-vr', function () {
```
Comment (fernandojsg) [inline src/ui2d.js]: can you remove this change? so it won't conflict with https://github.com/aframevr/a-saturday-night/pull/120

File: webpack.config.js [hunk 2]
```diff
@@ -11,5 +11,8 @@ module.exports = {
     path: __dirname,
     filename: 'build/build.js'
   },
-  plugins: PLUGINS
+  plugins: PLUGINS,
```
Comment (fernandojsg) [inline webpack.config.js]: already merged https://github.com/aframevr/a-saturday-night/pull/115

**Comment by fernandojsg** [comment]
I believe this one is good to go as long as the https://github.com/aframevr/aframe/pull/2939 will be merged so we could have both issues fixed.
Once merged we could keep the discussion on why that async issue when creating the elements and assigning the components. It looks weird to me too that it was working before without problem and now we need to attach these listeners.

File: src/ui2d.js (hunk 1 above)
Comment (caseyyee) [inline src/ui2d.js]: Sure thing

**Comment by caseyyee** [comment]
@fernandojsg ok, just fixed issues.  r?

**Comment by fernandojsg** [comment]
@caseyyee yes, just waiting for the PR https://github.com/aframevr/aframe/pull/2939 to be merged as without it I still get some exceptions

**Comment by fernandojsg** [comment]
@caseyyee https://github.com/aframevr/aframe/pull/2939 just got merged, could you rebase it so we could merge it

**Comment by fernandojsg** [comment]
@caseyyee nevermind, I'll just merge it and I'll rebase directly on master

**Comment by caseyyee** [comment]
Thanks for taking care of this!

**Comment by fernandojsg** [comment]
np!

**Pull request merged**